src/sqlfluff/rules/L016.py

```diff
@@ -520,8 +520,14 @@ def _eval(self, context: RuleContext) -> Optional[LintResult]:
             # these long lines will likely be single line Jinja comments.
             # They will remain as unfixable.
             if this_line[-1].type == "placeholder":
-                self.logger.info("Unfixable template segment: %s", this_line[-1])
-                return LintResult(anchor=context.segment, memory=memory)
+                if (
+                    this_line[-1].block_type != "comment"  # type: ignore
```

Comment:
Out of interest why does this need a `type: ignore`?

Reply:
mypy complains because `BaseSegment` doesn't have a `block_type` field. I could fix it properly by adding a `cast`, wasn't sure it was worth it. Thoughts?